src/main/scala/edu/berkeley/ce/rockslicing/RockSlicer.scala

```diff
@@ -10,17 +10,16 @@ object RockSlicer {
   def main(args: Array[String]) {
     val conf = new SparkConf().setAppName("SparkRocks")
     val sc = new SparkContext(conf)
-    val inputFile = args(0)
-    val numberSeedJoints = args(1).toInt
+    val inputs = CommandReader.argumentParser(args)
```

Comment:
Can we name this `args` or `arguments` instead? That will make it very clear what we're doing.